packages/firestore/test/unit/core/listen_sequence.test.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import { fail } from 'assert';
+import { ListenSequence } from '../../../src/core/listen_sequence';
+import { ListenSequenceNumber } from '../../../src/core/types';
+
+type SequenceNumberCallback = (sequenceNumber: ListenSequenceNumber) => void;
+
+describe('ListenSequence', () => {
+  it('writes the new sequence number to local storage', () => {
```

Comment:
Nit: These tests verify the interactions of the `syncer` interface, but don't use LocalStorage. Care to rename the descriptions? :)

Reply:
Done.